charts/firefly/README.md

```diff
@@ -441,10 +440,28 @@ annotation can be set on the `Ingress`:
 > it assumes the provided `hosts` must match the `tls[0].hosts` and that the secret is either pre-made or
 > provided by cert-manager.
 
-### Tokens via ERC1155 Connector
+### Tokens Connectors
+
+#### ERC1155
+
+By default, the chart comes with the [FireFly Tokens ERC1155 connector](https://github.com/hyperledger/firefly-tokens-erc1155)
```

Comment:
Mainly to match the `ff` CLI and the `firefly` E2E expectations but I have thought the same thing since I had to then disable it as part of the IT tests and the fact that it requires its own contract to be deployed.

Keeping it disabled by default for now is probably fine.